### ISSUE TITLE
AAAD not working

### ISSUE THREAD

**Issue by mcrazvan** [shmykelsa/AAAD#110]
Android Auto FULL version number - 6.6.612534
Phone model - Xiaomi M11Pro 5G
Android Version - 11
Working status - not working

AAAD installed ok, all apps installed ok(Carstream, Screen2Auto, AA Mirror...) No apps shown in Android Auto.

**Comment by Hav-Laz** [comment]
you need to download AA tweaker , then enable the third-party apps.
https://github.com/shmykelsa/AA-Tweaker

**Comment by shmykelsa** [comment]
> Android Auto FULL version number - 6.6.612534
> Phone model - Xiaomi M11Pro 5G
> Android Version - 11
> Working status - not working
> 
> AAAD installed ok, all apps installed ok(Carstream, Screen2Auto, AA Mirror...) No apps shown in Android Auto.

Could you please attach a screenshot as per these instructions:

- Head to phone settings 
- Select apps settings 
- Find any of the apps installed via AAAD
- Scroll down to the very bottom 
- Make a screenshot and attach it 

Thanks

**Comment by mcrazvan** [comment]
> 
> 
> you need to download AA tweaker , then enable the third-party apps.
> https://github.com/shmykelsa/AA-Tweaker

AAAD it's supossed to be used without ROOT. 
AA AIO TWEAKE it's requesting ROOT and my device is not ROOTED.

Thank's

**Comment by mcrazvan** [comment]
> * Head to phone settings
> 
> * Select apps settings
> 
> * Find any of the apps installed via AAAD
> 
> * Scroll down to the very bottom
> 
> * Make a screenshot and attach it

![AA Mirror](https://user-images.githubusercontent.com/88388919/128317309-201749ec-019d-43dc-a6cf-19b1ac15b8ac.jpg)

Attached, Thank's

**Comment by shmykelsa** [comment]
@mcrazvan by chance, have you just updated your phone's software?

**Comment by mcrazvan** [comment]
@shmykelsa yes, i have installed last MIUI 12.5.11 version but first try of install AAAD was in MIUI 12.5.10 with the same result. Security update is 01 June 2021.

**Comment by twlawrenceko** [comment]
@shmykelsa I can also confirm this is not working anymore. With the update on Samsung A12 Android 11 OneUI 3.1, fermata auto disappeared on Android auto from version 6.4 to 6.6.6. 

However, with Android 10. Fermata auto showed on Android auto with same AA version before Android 11 update

**Comment by shmykelsa** [comment]
> @shmykelsa yes, i have installed last MIUI 12.5.11 version but first try of install AAAD was in MIUI 12.5.10 with the same result. Security update is 01 June 2021.


After software update, it's recommend to uninstall the apps and install them again

**Comment by twlawrenceko** [comment]
> > @shmykelsa yes, i have installed last MIUI 12.5.11 version but first try of install AAAD was in MIUI 12.5.10 with the same result. Security update is 01 June 2021.
> 
> After software update, it's recommend to uninstall the apps and install them again

@shmykelsa  With the AAAD limitation of installing 1 app per month, I am not able to do so.

**Comment by mcrazvan** [comment]
> > @shmykelsa yes, i have installed last MIUI 12.5.11 version but first try of install AAAD was in MIUI 12.5.10 with the same result. Security update is 01 June 2021.
> 
> After software update, it's recommend to uninstall the apps and install them again

@shmykelsa I have PRO and I did but still not working.

**Comment by shoxter** [comment]
I have the same problem.

**Comment by 18clans** [comment]
Same thing happened to me too. I'm on One UI 3.1 and Andriod 11. Android Auto is on version 
7.1.614574-release.

Build number G973USQU6GUJ3
https://doc.samsungmobile.com/SM-G973U/SPR/doc.html

**Comment by 18clans** [comment]
If I remember correctly build number G973USQS6GUI1 was still working. Since the two updates no mirror app was showing up in Android Auto.

https://doc.samsungmobile.com/SM-G973U/SPR/doc.html

**Comment by SuperEggi88** [comment]
it seems i found a solution for the following problem.
After several months of perfectly working my Carstream and Fermata Auto disappeared.

I uninstalled the modified Android Auto App to Version 1.0, after installing the new AA app again i could find all my AAAD in the launcher and i was also able to see and use them in my car.....
 but after a while all app are disappeared.

Now my solution:
Delete all data and cache data in your Android Auto app, then uninstall all updates to Version 1.0.

Uninstall all Apps you have downloaded by AAAD, like Carstream, AAmirror....

Install the actual and needed Version from apkmirror or equal.
Then install by AAAD carstream.

This should work.

**Comment by Gizzy98** [comment]
So we need to remove any updates from AA to have this run now?
